### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by steccas** [goofball222/pritunl#11]
Hi, I'm on latest docker and docker-compose version on Manjaro Linux.

I need to build the Image because I'm testing it since I will need to deploy it on a raspberrypi.

Unfortunately the build fails reporting:

```
+ go get github.com/pritunl/pritunl-dns
package crypto/ed25519: unrecognized import path "crypto/ed25519" (import path does not begin with hostname)
The command '/bin/sh -c set -x  && apk update    && apk add -q --no-cache --virtual .build-deps         bzr curl gcc git libffi-dev linux-headers go       make musl-dev openssl-dev python2-dev py-pip     && apk add -q --no-cache         bash ca-certificates iptables ip6tables         openssl openvpn procps py2-setuptools         py2-dnspython tzdata wireguard-tools      && pip install --upgrade pip     && go get github.com/pritunl/pritunl-dns     && go get github.com/pritunl/pritunl-web     && cp /go/bin/* /usr/bin     && cd /tmp     && curl -sSL https://github.com/pritunl/pritunl/archive/${VERSION}.tar.gz -o /tmp/${VERSION}.tar.gz     && tar -zxf /tmp/${VERSION}.tar.gz     && cd /tmp/pritunl-${VERSION}     && python2 setup.py build     && pip install -r requirements.txt     && mkdir -p /var/lib/pritunl     && python2 setup.py install     && apk del -q --purge .build-deps     && rm -rf /go /root/.cache/* /tmp/* /var/cache/apk/*' returned a non-zero code: 1
ERROR: Service 'pritunl' failed to build
```

I'm afraid it depends on the too old version of go shipped with alpine 3.10; yet, many packages are available only in alpine 3.10

**Comment by steccas** [comment]
Can be solved with #12

**Comment by goofball222** [comment]
I was waiting for the latest version of pritunl to release, which supports python3. This issue is fixed as I can now switch to tracking alpine:latest again which has an updated version of go available. Should be an updated version published shortly.
